src/components/common-components/modal/error-modal/error.modal.list.item.component.tsx

```diff
@@ -3,17 +3,57 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 import React from "react";
+import { VerboseDate } from "../../../../common-models/month-info.model";
 import { ScheduleErrorMessageModel } from "../../../../common-models/schedule-error-message.model";
+import { TranslationHelper } from "../../../../helpers/translations.helper";
+import { useMonthInfo } from "../../../schedule-page/validation-drawer/use-verbose-dates";
 
 interface Options {
   error: ScheduleErrorMessageModel;
 }
 
 export default function ModalErrorListItem({ error }: Options): JSX.Element {
+  const { verboseDates, monthNumber } = useMonthInfo();
+  const mappedDays = verboseDates.map((d: VerboseDate) => d.date);
+  const monthStartIndex = verboseDates.findIndex((d: VerboseDate) => d.date === 1) ?? 0;
+
+  let currMonthGenetivus = "";
+  let prevMonthGenetivus = "";
+  if (monthNumber) {
+    currMonthGenetivus = `${TranslationHelper.polishMonthsGenetivus[monthNumber]}`;
+    if (monthNumber > 0) {
+      prevMonthGenetivus = `${TranslationHelper.polishMonthsGenetivus[monthNumber - 1]}`;
+    } else {
+      prevMonthGenetivus = `${TranslationHelper.polishMonthsGenetivus[monthNumber + 11]}`;
+    }
+  }
+
+  const errorDayIndex = error.day && error.day > 0 ? error.day : -1;
+  const errorDay = errorDayIndex ? mappedDays[errorDayIndex - 1] : "";
+  const monthName = errorDayIndex
+    ? errorDayIndex < monthStartIndex
+      ? prevMonthGenetivus
+      : currMonthGenetivus
+    : "";
+
+  const displayTitle = error.title && error.title !== "Nie rozpoznano błędu";
   return (
     <div className="error-list-item">
       <div className="red-rectangle" />
-      <div className="error-text">{error.message}</div>
+      <div className="error-modal">
+        {displayTitle && (
+          <div className="error-title">
+            <p className="error-title-content">
+              {error.title === "date" ? `${errorDay} ` + monthName : `${error.title}`}
+              {errorDayIndex > -1 && error.title !== "date" ? `, ${errorDay} ` + monthName : ``}
+            </p>
```

Comment:
Jeżeli do błędu przypisany jest dzień, to jest wyświetlany razem z tytułem:
![nieznana-wartosc-zmiany](https://user-images.githubusercontent.com/48785655/112428565-a245c380-8d3b-11eb-836b-03ac5353c06f.png)

Reply:
good